.github/workflows/test.yml

```diff
@@ -27,6 +27,22 @@ jobs:
       working-directory: ${{github.workspace}}/build
       run: ctest
 
+    - name: Install
+      working-directory: ${{github.workspace}}/build
```

Comment:
Do the install actions need the working directory to be set to `build` or would the top-level work?

Reply:
No, since the the `run` command uses absolute paths the `working-directory` is not necessary here; [removed](https://github.com/lcm-proj/lcm/compare/87930a1..df22576)